src/Libraries/PythonNodeModelsWpf/Resources/ICSharpCode.PythonBinding.Resources.Python.xshd

```diff
@@ -12,8 +12,8 @@
     <FoldMarker bold="false" italic="false" color="Gray" bgcolor="#242424" />
     <SelectedFoldLine bold="false" italic="false" color="Gray" />
     <EOLMarkers bold="false" italic="false" color="#999999" />
-    <SpaceMarkers bold="false" italic="false" color="#999999" />
-    <TabMarkers bold="false" italic="false" color="#999999" />
+    <SpaceMarkers bold="false" italic="false" color="#75715E" />
```

Comment:
if you search the avalonEdit code base for this element - it appears to be used absolutely no where... which is I guess why it does nothing.

Reply:
looks to me like they are hardcoded to white... https://github.com/icsharpcode/AvalonEdit/blob/28b887f78c821c7fede1d4fc461bde64f5f21bd1/ICSharpCode.AvalonEdit/Rendering/SingleCharacterElementGenerator.cs#L109

seems like a bug in avalonEdit maybe?